js/components/scrubberBar.js

```diff
@@ -398,17 +402,16 @@ var ScrubberBar = React.createClass({
       <div
         className="oo-scrubber-bar-container"
         ref="scrubberBarContainer"
-        onMouseEnter={scrubberBarMouseEnter}
-        onMouseLeave={scrubberBarMouseLeave}
-        onMouseMove={scrubberBarMouseMove}
-      >
+        onMouseOver={scrubberBarMouseOver}
+        onMouseOut={scrubberBarMouseOut}
+        onMouseLeave={this.handleScrubberBarMouseLeave}
```

Comment:
why this is the only one that is using "this."?

Reply:
It is the only one that was added recently. The other two were like that before, but I had changed them on the previous PR, and now I reverted them because the change had caused an issue. 
The old code was storing the handler in another variable for some reason, but this is unecessary